mautic/mautic
# Do not track Bots support

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | n
| New feature? | y
| Related user documentation PR URL | n
| Related developer documentation PR URL | n
| Issues addressed (#s or URLs) | n
| BC breaks? | n
| Deprecations? | n

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR added support for do not track list of bots. 
Bots list is from https://github.com/Thomas--F/BotTracker/blob/a7a2dcb1a510a958b58ff2a2028faf84027b5637/API.php#L120

#### Steps to test this PR:
1. Add tracking code to page
2. Verify from table ip_addresses if bot wasn't created as profile with history
For example google bot has ip details Mountain View, California.....
 Also you can check if IP is bot here http://bot.myip.ms/

![bots](https://cloud.githubusercontent.com/assets/462477/23226727/ea8d6852-f936-11e6-8c31-02e3765af79c.PNG)
